src/main/java/bio/terra/cli/service/WorkspaceManagerService.java

```diff
@@ -737,6 +744,24 @@ public RoleBindingList getRoles(UUID workspaceId) {
         "Error fetching users and their IAM roles for workspace");
   }
 
+  /**
+   * Call the Workspace Manager "/api/workspaces/v1/{workspaceId}/folders" endpoint to get a list of
+   * folders.
+   *
+   * @param workspaceId the workspace to query
+   * @return a list of folders in this workspace
+   */
+  public List<Folder> listFolders(UUID workspaceId) {
```

Comment:
i've been starting to use immutable collection in our repos.
Since this is a public method, can you use ImmutableList as the return type? Same with the method in Workspace.java.

https://g3doc.corp.google.com/third_party/java_src/google_common/current/java/com/google/common/collect/g3doc/immutable-collections.md?cl=head#why